### PR TITLE
PYIC-1186 Get VTM claim value from config and set on identity response

### PR DESCRIPTION
**Pull request by blakeyp**
### What changed

Get VTM claim value from config and set on identity response

### Why did it change

To ensure we're returning VTM in the identity claims

### Issue tracking
- [PYIC-1186](https://govukverify.atlassian.net/browse/PYIC-1186)

### CODE REVIEW

File: deploy/template.yaml [hunk 1]
```diff
@@ -545,6 +545,8 @@ Resources:
                 - 'kms:GenerateDataKey'
               Resource:
                 - !ImportValue AuditEventQueueEncryptionKeyArn
+        - SSMParameterReadPolicy:
+            ParameterName: !Sub ${Environment}/core/coreVtmClaim
```
Comment (dannyhayes-gds) [inline deploy/template.yaml]: I think the param name is slightly wrong here. You are missing the "self" bit in the middle.

```suggestion
            ParameterName: !Sub ${Environment}/core/self/coreVtmClaim
```

File: deploy/template.yaml [hunk 2]
```diff
@@ -787,4 +789,4 @@ Outputs:
     Description: Core Back External API Gateway ID
     Export:
       Name: !Sub "${AWS::StackName}-IPVCoreExternalAPIGatewayID"
-    Value: !Ref IPVCoreExternalAPI
+    Value: !Ref IPVCoreExternalAPI
```
Comment (dannyhayes-gds) [inline deploy/template.yaml]: Could you add an empty end of file line here

**Review by dannyhayes-gds — DISMISSED** [review]

**Review by kerrr — DISMISSED** [review]
Hold this a sec

**Review by dannyhayes-gds — APPROVED** [review]

**Review by kerrr — APPROVED** [review]
I believe Danny H is following up with a bit of a refactor of the CredientialIssuerStart lambda and test to make some of these changes unnecessary. But good to merge now. 🚀

**Pull request merged**